server/app/forms/ProgramForm.java

```diff
@@ -5,13 +5,16 @@
 
 /** Form for updating name and description of a program. */
 public final class ProgramForm {
+  public static final String NO_EMAIL_NOTIFICATIONS_OPTION = "NO_EMAIL_NOTIFICATIONS";
+
   private String adminName;
   private String adminDescription;
   private String localizedDisplayName;
   private String localizedDisplayDescription;
   private String localizedConfirmationMessage;
   private String externalLink;
   private String displayMode;
+  private List<String> notificationPreferences;
```

Comment:
**thought (non-blocking):** Out of context this feels like over-engineering, but I'm guessing you chose this option intentionally with the future roadmap in mind such that YAGN hopefully won't apply

Reply:
Yeah, using an array instead of just a boolean was _slightly_ more work, but seemed worth the tradeoff in the event that we want to add other notification types, like a daily batched email, in the future.